cmd/wallet-web/src/components/CredentialPreview/CredentialPreview.vue

```diff
@@ -8,36 +8,36 @@
   <router-link
     :class="[
       `group outline-none inline-flex items-center rounded-xl py-6 pl-5 pr-3 text-sm md:text-base font-bold border w-full h-20 md:h-24 focus-within:ring-2 focus-within:ring-offset-2 credentialPreviewContainer`,
-      brandColor.length
-        ? `bg-gradient-${brandColor} border-neutrals-black border-opacity-10 focus-within:ring-primary-${brandColor}`
-        : `bg-neutrals-white border-neutrals-thistle hover:border-neutrals-chatelle focus-within:ring-neutrals-victorianPewter`,
+      styles.background.color !== '#fff'
```

Comment:
This is as per the existing implementation https://github.com/birtony/wallet/blob/1b4f085e1c051d832e031c9f903bf593df0317de/cmd/wallet-web/src/components/CredentialDetails/Banner.vue#L24 This basically checks if styles background color is provided otherwise shows default color.

Reply:
Ok - probably better to add a comment to describe the reason.

Reply:
added here https://github.com/trustbloc/wallet/blob/ffe9a217d9205f3f9ac882b2319b3ecf70369a73/cmd/wallet-web/src/components/CredentialPreview/CredentialPreview.vue#L8